roles/kubernetes/master/templates/kubeadm-config.v1beta2.yaml.j2

```diff
@@ -19,7 +19,7 @@ nodeRegistration:
 {% if inventory_hostname in groups['kube-master'] and inventory_hostname not in groups['kube-node'] %}
   taints:
   - effect: NoSchedule
-    key: node-role.kubernetes.io/master
+    key: node-role.kubernetes.io/control-plane
```

Comment:
noting that existing user workloads that do not tolerate this new taint will fail to schedule.

Reply:
Yup that's why I flag the PR as draft, I have some testing to do and will probably only change label for now